vimrc

```diff
@@ -30,6 +30,24 @@
 " Note: To install a plugin, put it in pack/foo/start/<plugin name>.
 " See <https://shapeshed.com/vim-packages/> for details.
 
+set nocompatible
+filetype off
+
+call plug#begin('~/.vim/plugged')
+
+Plug 'epmatsw/ag.vim'
+Plug 'scrooloose/nerdtree', { 'on':  'NERDTreeToggle' }
+Plug 'ctrlpvim/ctrlp.vim'
+Plug 'tpope/vim-surround'
+Plug 'tpope/vim-unimpaired'
+Plug 'editorconfig/editorconfig-vim'
```

Comment:
Looks like we already have editorconfig via a native vim package: https://github.com/seattle-beach/vim-config/tree/master/pack/foo/start/editorconfig

Reply:
cool, I think I verified I'm still getting editorconfig stuff after removing that.